DOCS.md

```diff
@@ -1,31 +1,380 @@
+# drone-gke Usage
+
 Use this plugin to deploy Docker images to [Google Container Engine (GKE)][gke].
 
 [gke]: https://cloud.google.com/container-engine/
 
-## Overview
+## API Reference
 
 The following parameters are used to configure this plugin:
 
-* `image` - this plugin's Docker image
-* *optional* `project` - project of the container cluster (defaults to inferring from the service account `token` credential)
-* *optional* `zone` - zone of the container cluster
-* *optional* `region` - region of the container cluster
-* `cluster` - name of the container cluster
-* *optional* `namespace` - Kubernetes namespace to operate in (defaults to `default`)
-* *optional* `template` - Kubernetes manifest template (defaults to `.kube.yml`)
-* *optional* `secret_template` - Kubernetes [_Secret_ resource](http://kubernetes.io/docs/user-guide/secrets/) manifest template (defaults to `.kube.sec.yml`)
-* *optional* `wait_deployments` - list of Deployments to wait for successful rollout, using `kubectl rollout status`
-* *optional* `wait_seconds` - if `wait_deployments` is set, number of seconds to wait before failing the build
-* `vars` - variables to use in `template` and `secret_template` (see [below](#available-vars) for details)
-* `secrets` - credential and variables to use in `secret_template` (see [below](#secrets) for details)
-* *optional* `expand_env_vars` - expand environment variables for values in `vars` for reference (defaults to `false`)
+### `image`
```

Comment:
This is very verbose, I'd like to stick to the original format, or in the drone plugins registry format:
- http://plugins.drone.io/drone-plugins/drone-docker/
- https://github.com/jmccann/drone-terraform/blob/master/DOCS.md

Reply:
is there any automation built around those formats? happy to use a less verbose alternative, just thinking it would be helpful to enable deep linking directly to a specific parameter's documentation..

Reply:
AFAIK no there isn't.

Reply:
hm if there isn't any automation built around the previous format or the other formats used, is there a functional downside to these changes (aside from the more verbose markdown source)?

in addition to the deep links, i think some folks might find the in-context `example`s helpful.. for a concrete case, i myself had to view the source of the plugin to find out how to use the `wait_deployments` option (do i provide a string? if so, separated by what character? there aren't any other options that accept an array of values so it wasn't clear that this one was expected to be provided as such) .. for another, i wasn't aware that values within `vars` could be any arbitrary value (e.g., could include more than the primitive string / int types).. that might have been more obvious if the type info was included in the docs for each option